script.js

```diff
@@ -65,3 +65,18 @@ function updateUncheckedTodoCount() {
   const todosLeftElement = document.getElementById("todos-left");
   todosLeftElement.textContent = `Todos left unchecked: ${uncheckedTodos}`;
 }
+
+function setFilter(filterValue) {
+  const filterRadioButton = document.getElementById(
+    `option${filterValue === "all" ? 1 : filterValue === "completed" ? 3 : 2}`
+  );
+  if (filterRadioButton) {
+    filterRadioButton.checked = true;
+    state.filter = filterValue;
```

Comment:
This assignment here is kind of strange. I'd consider setting the state first and then call something to render reflecting that change, right?